tools/expotools/src/Changelogs.ts

```diff
@@ -63,14 +79,107 @@ export class Changelog {
     this.filePath = filePath;
   }
 
+  /**
+   * Resolves to `true` if changelog file exists, `false` otherwise.
+   */
+  async fileExistsAsync(): Promise<boolean> {
+    return await fs.pathExists(this.filePath);
+  }
+
+  /**
+   * Lexifies changelog content and returns resulting tokens.
+   */
   async getTokensAsync(): Promise<Markdown.Tokens> {
     if (!this.tokens) {
-      await fs.access(this.filePath, fs.constants.R_OK);
-      this.tokens = Markdown.lexify(await fs.readFile(this.filePath, 'utf8'));
+      try {
+        await fs.access(this.filePath, fs.constants.R_OK);
```

Comment:
```suggestion
```
Is it necessary to have it here?
I mean when the `file` is not `readable` then error is thrown and in the next line `fs.readFile` will throw an error if the file is not `readable` as well. 🤔